graylog2-server/src/main/java/org/graylog2/contentpacks/model/ContentPackV1.java

```diff
@@ -84,10 +83,18 @@
 
     public static Builder builder() {
         return new AutoValue_ContentPackV1.Builder()
-                .requires(ImmutableSet.of())
                 .parameters(ImmutableSet.of());
     }
 
+    public Set<Constraint> constraints() {
+        return entities().asList().stream()
```

Comment:
What's the reason for using `.asList()` here? Doesn't the following just work as well?
```suggestion
        return entities().stream()
```